haystack/retriever/dense.py

```diff
@@ -118,7 +118,7 @@ def __init__(self,
                                                  metric="text_similarity_metric",
                                                  embed_title=self.embed_title,
                                                  num_hard_negatives=0,
-                                                 num_negatives=0)
+                                                 num_positives=0)
```

Comment:
Can you please adjust the default to `1`? This would be more meaningful. Thx!

Reply:
done